ext/ffi/lib.rs

```diff
@@ -269,6 +270,63 @@ struct FfiLoadArgs {
   symbols: HashMap<String, ForeignFunction>,
 }
 
+// `path` is only used on Windows.
+#[allow(unused_variables)]
+fn format_error(e: dlopen::Error, path: String) -> String {
+  match e {
+    #[cfg(target_os = "windows")]
+    // This calls FormatMessageW with library path
+    // as replacement for the insert sequences.
+    // Unlike libstd which passes the FORMAT_MESSAGE_IGNORE_INSERTS
+    // flag without any arguments.
+    //
+    // https://github.com/denoland/deno/issues/11632
+    dlopen::Error::OpeningLibraryError(e) => {
```

Comment:
Would it be useful to do this for other kinds of errors?

Reply:
`OpeningLibraryError` is the only possible error when calling `Deno.dlopen`. We probably won't even need to map other errors if they don't have any insert sequences. Mapping them right now will lead to weird issues without the presence of the `FORMAT_MESSAGE_IGNORE_INSERTS` flag and we will have to maintain a list of Win32 errors that need that flag.